treestats_paper.tex

```diff
@@ -353,12 +353,13 @@ \section*{Framework and statistics}
 \subsection*{Node statistics}
 
 Perhaps the most natural way to summarize weights on the tree
-is simply to examine the values at each node.
-This would allow us, for instance, to
-count the number of samples that inherit from each node in the tree.
+is to examine the values at each node.
+\krt{I hope I didn't make this bit incorrect?  I found myself having to reread it several times.}
```

Comment:
I think this isn't quite right @molpopgen --- isn't the point that the simplest f we can have to just return the weights directly (lambda x: x)? The resulting statistic then tells us how many samples inherit from each node in the tree.

Reply:
Yeah, that's right.  I'll try to tweak this further.  The description is quite abstract, I think, and could perhaps be simpler.

Reply:
The i.e. are Latin, hence the emph.  But yeah, that'd probably be done by the journal.

Reply:
I mostly reverted this change & tried to make it a bit more explicit.